services/bettersocial/fcmToken/sendGroupChatNotification.js

```diff
@@ -0,0 +1,26 @@
+const {messaging} = require('firebase-admin');
+const FcmTokenFunction = require('../../../databases/functions/fcmToken');
+const {FcmToken} = require('../../../databases/models');
+
+const sendGroupChatNotification = async (userTargetId, message) => {
+  const userTargetToken = await FcmTokenFunction.findTokenByUserId(FcmToken, userTargetId);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `sendGroupChatNotification` is asynchronous but it doesn't seem to return anything. If the intention is to wait for the notification to be sent before proceeding, you should return the promise from `messaging().sendToDevice()`. This way, the caller can await the completion of the notification sending.



<!-- This is an auto-generated comment by OSS CodeRabbit -->